docs/community/release_guide.md

```diff
@@ -21,7 +21,7 @@ Celeborn Release Guide
 
 ## Introduction
 
-The Apache Celeborn (Incubating) project periodically declares and publishes releases. A release is one or more packages
+The Apache Celeborn project periodically declares and publishes releases. A release is one or more packages
```

Comment:
This release guide can be changed into a TLP version. You can refer to [Kybbui's release guide](https://github.com/apache/kyuubi/blob/master/docs/contributing/code/release.md).